config/init.lua

```diff
@@ -104,6 +105,10 @@ for _, key in pairs(keys) do
     config.register_key(key)
 end
 
+function cleanup_background()
+  os.execute("pkill way-cooler-bg")
+end
+
 
 -- Execute some code after Way Cooler is finished initializing
 function way_cooler_init()
```

Comment:
It was probably added in the same PR as when I split up the background program, because that was the point we needed `way_cooler_init`.